client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -390,9 +390,15 @@ function getDisplayModeParamFromCart( cart: ResponseCart | undefined ): Record<
 	if ( cart && hasConciergeSession( cart ) ) {
 		return { d: 'concierge' };
 	}
+
 	if ( cart && hasTrafficGuide( cart ) ) {
 		return { d: 'traffic-guide' };
 	}
+
+	if ( cart ) {
+		return { d: 'purchase-success' };
```

Comment:
As already mentioned, since this function is 100% covered by tests, it would be great if you could add a test to cover this new behavior!